### ISSUE TITLE
[Raisinbread] No data for help editor module?

### ISSUE THREAD

**Issue by johnsaigle** [aces/Loris#5691]
It doesn't seem to be possible to edit any help files using the Raisinbread dataset. The Test VM, demo, and my sandbox contain no data.

How is this module being tested?

**Comment by maltheism** [comment]
@johnsaigle I'm debugging this issue and so far I think inside `help_editor.class.inc` the query for building the table is not returning anything.
```
SELECT helpChild.helpID as helpID,
                    helpChild.topic as Topic,
                    helpChild.parentID as Parent_ID,
                    helpParent.parentID as ParentTopic_ID,
                    helpParent.topic as Parent_Topic,
                    helpChild.content as Content
             FROM help helpChild
             LEFT JOIN help helpParent
               ON (helpChild.parentID = helpParent.helpID)
             WHERE helpChild.hash IS NOT NULL
             AND helpChild.topic IS NOT NULL
             ORDER BY helpChild.helpID
```
I'm not finding anything in the database for the help_editor query. @ridz1208 would you know if rasinbread doesn't populate it?

**Comment by ridz1208** [comment]
i thought that was replaced with the files on the filesystem ??

**Comment by johnsaigle** [comment]
If so then there shouldn't be a query, I think? Also if that is the case then the file system ones aren't loading. Either way the module is empty.

**Comment by christinerogers** [comment]
@ridz1208 I dont' see a fix @AlexandraLivadas  can implement here, to help with testing next round.  shall we close this, while the future of the module is hanging in the balance?

**Comment by ridz1208** [comment]
i think we could add a help document for the BMI ? then we put this to rest once and for all

**Comment by christinerogers** [comment]
ok that sounds worth a shot.  
@AlexandraLivadas let's give this a try -- 
Take a few minutes to draft some general "How to fill out the BMI" instructions, make a PR to add them in the right place, also post a screenshot here of how it renders in the dropdown Help when you're looking at the BMI data entry form. We'll refine it from there.

**Comment by AlexandraLivadas** [comment]
Resolved by #6907
